orchardcore.ruleset

```diff
@@ -2,15 +2,20 @@
 <RuleSet Name="Orchard Core C# rules" ToolsVersion="17.0">
   <Include Path="general.ruleset" Action="Default" />
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA1014" Action="None" />
     <Rule Id="CA1710" Action="Hidden" />
     <Rule Id="CA1711" Action="Hidden" />
     <Rule Id="CA1716" Action="Hidden" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp" RuleNamespace="Microsoft.CodeAnalysis.CSharp">
+    <Rule Id="CS1591" Action="None" />
+  </Rules>
   <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.NetAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.CSharp.NetAnalyzers">
     <Rule Id="CA1019" Action="Hidden" />
     <Rule Id="CA1812" Action="Hidden" />
   </Rules>
   <Rules AnalyzerId="Microsoft.CodeAnalysis.NetAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.NetAnalyzers">
+    <Rule Id="CA1014" Action="None" />
```

Comment:
The same code in 4 places - really? 🧷

Reply:
Yes, it's depressing. (What does safety pin emoji mean here?)

Reply:
lol 😹